core/src/main/kotlin/org/opensearch/alerting/core/ScheduledJobIndices.kt

```diff
@@ -38,7 +38,14 @@ class ScheduledJobIndices(private val client: AdminClient, private val clusterSe
         if (!scheduledJobIndexExists()) {
             var indexRequest = CreateIndexRequest(ScheduledJob.SCHEDULED_JOBS_INDEX)
                 .mapping(scheduledJobMappings())
-                .settings(Settings.builder().put("index.hidden", true).build())
+                .settings(
```

Comment:
Do you want to add some comments (documentation) for rationale behind the primary and replica count?